OAuth/RequestDataStorage/SessionStorage.php

```diff
@@ -25,28 +25,33 @@
 class SessionStorage implements RequestDataStorageInterface
 {
     /**
-     * @var SessionInterface
+     * @var RequestStack
      */
-    private $session;
+    private $requestStack;
 
-    public function __construct(SessionInterface $session)
+    public function __construct(RequestStack $requestStack)
```

Comment:
This would be a BC break if someone extends `SessionStorage`.

Reply:
I think we can not it in `CHANELOG.md`, we don't expect that ppl extend it instead of using interface.

Reply:
Should we make it final then? We could add `@final` to prevent BC break.

Reply:
Added note + marked as `@final since 1.4`.